<launch>
    
    <!--Joy Node-->
    <node pkg="joy" type="joy_node" name="dumbobot_joy_node" respawn="true">
        <param name="dev" type="string" value="/dev/input/js0" />
        <param name="deadzone" value="0.12" />
    </node>

    <!--Axes-->
    <param name="axis_linear" value="1" type="int"/>
    <param name="axis_angular" value="3" type="int"/>
    <param name="button_deadman_switch" value="2" type="int"/>
    <param name="scale_linear" value="0.60" type="double"/>
    <param name="scale_angular" value="1.78" type="double"/>
    <param name="twist_pub_topic" value="/dumbobot/cmd_vel" type="string"/>
    <!--<param name="twist_pub_topic" value="/ros_dumbobot/cmd_vel" type="string"/>-->
    <param name="joystick_sub_topic" value="joy" type="string"/>
    <param name="auto_stop_pub_topic" value="/dumbobot/move_base/cancel" type="string"/>

    <!--Dumbobot Joy-->
    <node pkg="dumbobot_teleop" type="dumbobot_teleop_joy_node" name="dumbobot_teleop_joy_node">
    </node>

</launch>